src/cpp/src/llm_pipeline.cpp

```diff
@@ -96,6 +97,9 @@ class StatefulLLMPipeline final : public LLMPipelineImplBase {
         // If eos_token_id was not provided, take value
         if (m_generation_config.eos_token_id == -1)
             m_generation_config.set_eos_token_id(m_tokenizer.get_eos_token_id());
+
+        m_sampler = Sampler(m_tokenizer);
```

Comment:
You can move this to member initializer list for both pipelines

Reply:
I'm not sure I understand right what do you mean, do you mean moved it here https://github.com/openvinotoolkit/openvino.genai/blob/af3d3e12f5f1bc6b8c0b6720c93327313f630fa4/src/cpp/src/llm_pipeline.cpp#L48 ?

Reply:
No. There's a special initialization form in C++ called `member initializer list`. It should look like
```cpp
    StatefulLLMPipeline(
        const std::shared_ptr<ov::Model>& model,
        const ov::genai::Tokenizer& tokenizer,
        const std::string& device,
        const ov::AnyMap& config,
        const ov::genai::GenerationConfig& generation_config
    ) : 
        LLMPipelineImplBase(tokenizer, generation_config),
        m_sampler(m_tokenizer) {
```